src/components/LanguageSwitch/LanguageSwitch.tsx

```diff
@@ -139,10 +144,7 @@ const LanguageSwitch = () => {
                     .map(language => {
                       const { icon, translation } = getTranslatedCurrentLanguageDetails(t, language);
                       return (
-                        <StyledMenuItem
-                          key={language}
-                          onClick={handleSwitchLanguage(language)}
-                          selected={userLanguage === translation}>
+                        <StyledMenuItem key={language} onClick={handleSwitchLanguage(language)} selected={userLanguage === translation}>
```

Comment:
It seems linter is yelling, here, could you check that?

Reply:
fixed